Sources/Sentry/SentryFileManager.m

```diff
@@ -787,7 +811,39 @@ - (void)createPathsWithOptions:(SentryOptions *)options
 
     NSError *error;
     SENTRY_CASSERT([fm removeItemAtPath:path error:&error],
-        @"Failed to remove launch profile marker file: %@", error);
+        @"Failed to remove launch profile config file: %@", error);
+}
+
+void
+removeAppLaunchProfilingConfigBackupFile(void)
+{
+    NSFileManager *fm = [NSFileManager defaultManager];
+    NSString *path = launchProfileConfigBackupFileURL().path;
+    if (![fm fileExistsAtPath:path]) {
+        return;
+    }
+
+    NSError *error;
+    SENTRY_CASSERT([fm removeItemAtPath:path error:&error],
+        @"Failed to remove launch profile config file: %@", error);
+}
+
+void
+saveAppLaunchProfilingConfigFile(void)
+{
+    NSFileManager *fm = [NSFileManager defaultManager];
+    NSString *path = launchProfileConfigFileURL().path;
+    if (!SENTRY_CASSERT_RETURN([fm fileExistsAtPath:path],
+            @"Expect to have a current launch profile config to use for subsequent transaction.")) {
+        return;
+    }
+
+    NSError *error;
+    SENTRY_CASSERT([fm moveItemAtPath:path
```

Comment:
`h`: What if the backup file already exists? This method then throws an error. This could happen if an app crashes during the start process. I think we should overwrite the existing config file instead. A test for that in `SentryFileManagerTests` would be great.

Reply:
Yep, i think overwriting is the way to go as well.